README.md

```diff
@@ -130,6 +130,14 @@ To just get a list of commits contained in rpi-update since you last updated, ru
 
 This won't update your firmware
 
+#### `QUIET_MODE`
+
+Makes curl quiet.
+
+    sudo QUIET_MODE=1 rpi-update
+
+This only changes the verbosity of curl.
```

Comment:
If it _only_ affects the curl quietness, maybe it should be `QUIET_CURL_MODE` instead? :wink:

Reply:
It's been so long since I submitted this (Dec 31, 2016) that I'd forgotten all about it...  

Thanks!